### DISCUSSION TITLE
Collection Types

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#10] [Progress Tracker]
Topics Covered:
👉 What is a List.
👉 Declaring a Dictionary.
👉 List item count
👉 List vs Array.
👉 Adding elements using Add and AddRange methods.
👉 Updating elements from List and Dictionary.
👉 Removing elements from List and Dictionary.
👉 Looping over a using for and foreach loop.
👉 Looping over dictionary using foreach loop.
👉 Accessing keys and values of the dictionary.
👉 Creating a List of Dictionaries.
👉 Using other methods and properties like Count, Min, Max, Clear, Remove, RemoveAt etc...

**Comment by abijithks266** [comment]
completed

**Comment by ManikuttyBabu** [comment]
completed

**Comment by PEFathimaManfa** [comment]
completed

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by Arun-babumon** [comment]
Completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by Nandana-04** [comment]
Completed

**Comment by Jiby-Joseph-123** [comment]
Completed.

**Comment by riya45810** [comment]
Completed.

**Comment by akhilamp123** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
